src/pages/projectSettings/tabs/PeriodicBuildsTab/getFormSchema.ts

```diff
@@ -0,0 +1,126 @@
+import { Field } from "@rjsf/core";
+import { SpruceFormProps } from "components/SpruceForm";
+import widgets from "components/SpruceForm/Widgets";
+import { timeZones } from "constants/fieldMaps";
+import { form, ProjectType } from "../utils";
+
+const { overrideRadioBox } = form;
+
+export const getFormSchema = (
```

Comment:
I'm pretty sure I've configured this correctly so I'm not sure why I'm not able to save the page.
<img width="890" alt="image" src="https://user-images.githubusercontent.com/26798134/161626397-17925d4d-acd4-4566-a10e-f1a058fe76c1.png">